README.md

```diff
@@ -2,10 +2,36 @@
-| This is a small frontend to trial user flows for a future CRA service that will help Canadians receive the benefits to which they are entitled.                                                            | Voici une petite interface qui nous sert à tester des parcours utilisateur pour un futur service de l'ARC qui aidera les Canadiens à recevoir les prestations auxquelles ils ont droit.                                      |
-| It’s a server-side [express](https://expressjs.com/) application using [Pug](https://pugjs.org/api/getting-started.html) templating on the server and schnazzy [SCSS](https://sass-lang.com/) stylesheets. | Il s'agit d'une application [express](https://expressjs.com/) qui utilise les modèles [Pug](https://pugjs.org/api/getting-started.html) sur le serveur ainsi que des chics feuilles de style [SCSS](https://sass-lang.com/). |
+This is a demo of a service to make tax filing faster and easier for a preselected group of eligible low-income Canadians. This service was designed and developed by the [Canadian Digital Service (CDS)](https://digital.canada.ca/) in partnership with the [Canada Revenue Agency (CRA)](https://www.canada.ca/en/revenue-agency.html).
+
+Much of the information needed to file a tax return is already known to CRA. At the same time, low-confidence tax filers are very worried about making mistakes when submitting a tax return. The concept behind this demo application is to provide a wizard-like tax filing experience: presenting the tax filer with information that CRA already has about them and asking them to confirm it rather than having them type it in.
+
+While this repository is no longer actively maintained, the code is open and available for demonstration purposes or reuse. In addition to an end-to-end flow, it is an example of best-practice development for an API-driven cloud-native frontend application.
+
+## Table of contents
+
+- [Technical overview](#technical-overview)
+  - [Use of third-party services](#use-of-third-party-services)
+  - [Automated tests](#automated-tests-)
+- [Getting started](#getting-started-npm)
+  - [Build and run](#build-and-run)
+  - [Run tests](#run-tests)
+- [Using Docker](#using-docker)
+- [Deploying the app](#deploying-the-app)
+
+## Technical overview
+
+The Claim Tax Benefits (CTB) application a server-side [express](https://expressjs.com/) application using [Pug](https://pugjs.org/api/getting-started.html) templating on the server and schnazzy [SCSS](https://sass-lang.com/) stylesheets. We used [the express generator](https://expressjs.com/en/starter/generator.html) for application scaffolding, so the structure of the application comes from there. It assumes the existence of a backend API (to receive user data) and a cloud database (for access codes), although these are stubbed out in the app so that it can run as a standalone application for development purposes.
```

Comment:
missing an 'is' here, or colon depending on which way you wanna go
`The Claim Tax Benefits (CTB) application a server-side `